### PR TITLE
Export to excel redo

### PR DESCRIPTION
**Pull request by bhalonen**
## Exports faceforce to excel using json format

Related PR's
- [ExcelMakerConsole](https://github.com/relinc/ExcelMakerConsole/pull/2)

## How to Test
Export to excel on windows platform with and without face forces involved.

### CODE REVIEW

File: Viewer/src/net/relinc/viewer/GUI/ExportGUI.java [hunk 1]
```diff
@@ -450,77 +441,127 @@ private void writeCSVFile(File file){
...
+		return description;
+	}
 	private File writeConsoleExcelFileMakerJobFile(String path, int pointsToKeep) {
 		File jobFile =new File(SPSettings.applicationSupportDirectory + "/RELFX/SUREPulse/JobFile");
 		if(jobFile.exists())
 			SPOperations.deleteFolder(jobFile);
 		jobFile.mkdir();
+		JSONObject excelJobDescription = new JSONObject();
+        //Check to see if faceforces exist in all samples
+        boolean faceForcePresent = isFaceForcePresent();
```
Comment (markhalonen) [inline Viewer/src/net/relinc/viewer/GUI/ExportGUI.java]: spaces mixed with tabs

**Comment by markhalonen** [comment]
Tried testing but getting this error
```
Exception in thread "JavaFX Application Thread" java.lang.ArrayIndexOutOfBoundsException: 13488
	at net.relinc.viewer.GUI.ChartsGUI.getFaceForceTimeChart(ChartsGUI.java:572)
	at net.relinc.viewer.GUI.HomeController.getFaceForceTimeChart(HomeController.java:1858)
	at net.relinc.viewer.GUI.HomeController.getChart(HomeController.java:1620)
	at net.relinc.viewer.GUI.HomeController.renderCharts(HomeController.java:1127)
	at net.relinc.viewer.GUI.HomeController$10.onChanged(HomeController.java:584)
```
with this sample (attached) 

[B03.samcomp.zip](https://github.com/relinc/SurePulse/files/2981478/B03.samcomp.zip)

probably not due to this PR, assuming we should make another issue for this. I suspect my big refactor for multiple results may have caused this.

**Comment by bhalonen** [comment]
I looked into the sample and I found some odd things.
First, I got this error:
```WARNING: Loading FXML document with JavaFX API of version 9 by JavaFX runtime of version 8.0.51
java.io.FileNotFoundException: /Users/brent/Library/Application Support/RELFX/SUREPulse/tmp/317e183c-78e1-495e-81a3-2f12606afe94/Parameters.json (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at java.io.FileReader.<init>(FileReader.java:58)
	at net.relinc.libraries.staticClasses.SPOperations.readStringFromFile(SPOperations.java:142)
```
The folder in question has a zipped sample file:
```Brents-MBP-3:SurePulse brent$ ls /Users/brent/Library/Application\ Support/RELFX/SUREPulse/tmp/317e183c-78e1-495e-81a3-2f12606afe94/
B03.samcomp
__MACOSX```

Which is kinda confusing.

**Comment by markhalonen** [comment]
I think the ExcelMakerConsole.exe should get updated here after the changes in that repo

**Comment by markhalonen** [comment]
I'm not seeing the Face Forces graphed on the "Summary" excel sheet, only in the individual group excel sheets
image

**Comment by markhalonen** [comment]
Looking solid. JSON conversion definitely is a big improvement.

I think the remaining things to address are
- [ ] mix of tabs and spaces
- [ ] no face force graphs in the summary excel sheet
- [ ] update with new ExcelMakerConsole.exe and test

**Comment by bhalonen** [comment]
I actually wanted to avoid the summary graphing of the face forces. I think it would just be incredibly confusing, way too much going on.

**Comment by markhalonen** [comment]
Ok fine by me, just would be more consistent. I really don't even know if users export in this format anymore

**Comment by markhalonen** [comment]
I guess ExcelMakerConsole.exe will theoretically be exactly the same, just changed some dependency stuff

**Comment by bhalonen** [comment]
Pry still good to get a rebuild for sanity sakes.

**Pull request merged**